packages/wasm/crate/src/database/indexed_db.rs

```diff
@@ -39,10 +40,12 @@ async fn mock_test_database(mut db_req: OpenDbRequest) -> OpenDbRequest {
                 .create_object_store_with_params("SPENDABLE_NOTES", &note_object_store_params)?;
 
             let nullifier_key: JsValue = serde_wasm_bindgen::to_value("nullifier.inner")?;
+            let params = web_sys::IdbIndexParameters::new();
+            params.set_unique(false);
             note_object_store.create_index_with_params(
                 "nullifier",
                 &IdbKeyPath::new(nullifier_key),
-                web_sys::IdbIndexParameters::new().unique(false),
```

Comment:
Clippy flagged `unique()` being a deprecated API and that `set_unique()` should be used instead